projects/plugins/jetpack/extensions/shared/memberships/settings.js

```diff
@@ -85,6 +98,49 @@ function NewsletterAccessSetupNudge( { stripeConnectUrl, isStripeConnected, hasN
 	}
 }
 
+function TierSelector( { onChange } ) {
+	// TODO: figure out how to handle different currencies
+	const products = useSelect( select => select( membershipProductsStore ).getProducts() )
+		.filter( product => product.subscribe_as_site_subscriber && product.interval === '1 month' )
+		.sort( ( p1, p2 ) => Number( p2.price ) - Number( p1.price ) );
+
+	// Find the current tier meta
+	const postType = useSelect( select => select( editorStore ).getCurrentPostType(), [] );
+	// Destructure the tierId from the meta (set tierId using the META_NAME_FOR_POST_TIER_ID_SETTINGS constant)
+	const [ { [ META_NAME_FOR_POST_TIER_ID_SETTINGS ]: tierId } ] = useEntityProp(
+		'postType',
+		postType,
+		'meta'
+	);
+
+	// Tiers don't apply if less than 2 products (this is called here because
+	// the hooks have to run before any early returns)
+	if ( products.length < 2 ) {
+		return;
+	}
+
+	return (
+		<div className="jetpack-editor-post-tiers">
+			<RadioControl
+				label={ __( 'Choose Newsletter Tier', 'jetpack' ) }
+				hideLabelFromVision={ true }
+				selected={ Number( tierId ) }
+				options={ products.map( product => {
+					/* Translators: %s is the tier label created by site user */
+					const label = sprintf( __( '%s subscribers', 'jetpack' ), product.title );
```

Comment:
You'll want to move the translator comment right above the translatable, otherwise the linting tools are not going to be happy:

```suggestion
					const label = sprintf(
						/* Translators: %s is the tier label created by site user */
						__( '%s subscribers', 'jetpack' ),
						product.title
					);
```

Reply:
@n3f